src/avt/VisWindow/Colleagues/avtAnnotationColleague.C

```diff
@@ -45,3 +47,29 @@ avtAnnotationColleague::avtAnnotationColleague(VisWindowColleagueProxy &m) :
 avtAnnotationColleague::~avtAnnotationColleague()
 {
 }
+
+// ****************************************************************************
+// Method: avtAnnotationColleague::SetTimeScaleAndOffset
+//
+// Purpose: 
+//   Sets the scale and offset that will be applied to the time.
+//
+// Arguments:
+//   scale  : Multiplier for the time.
+//   offset : Offset that will be added to the time.
+//
+// Programmer: Brad Whitlock
+// Creation:   Mon Mar  2 14:13:57 PST 2009
+//
+// Modifications:
+//   
```

Comment:
Move the blank comment line below the comment.

Reply:
FYI...it is possible to do multi-line suggestions for these cases. To do multi-line suggestions, you select the *first* line of the suggested change and then extend-select to the last line of the suggested change and then hit the suggestion button.